### PR TITLE
Minor cleanup in CreateChildBone action

### PR DESCRIPTION
**Pull request by rodolforg**

### CODE REVIEW

File: synfig-studio/src/synfigapp/actions/valuedesccreatechildbone.cpp [hunk 1]
```diff
@@ -216,7 +215,9 @@ Action::ValueDescCreateChildBone::prepare()
 
 	Action::Handle action;
 
-	action = ValueNodeStaticListInsert::create();
+	action = Action::create("ValueNodeStaticListInsert");
```
Comment (ice0) [inline synfig-studio/src/synfigapp/actions/valuedesccreatechildbone.cpp:218]: Is there any reason to use this approach?

Comment (rodolforg) [reply synfig-studio/src/synfigapp/actions/valuedesccreatechildbone.cpp:218]: Just for avoiding `#include "valuenodestaticlistinsert.h"`

Comment (ice0) [reply synfig-studio/src/synfigapp/actions/valuedesccreatechildbone.cpp:218]: Ok

**Review by ice0 — APPROVED** [review]

**Comment by ice0** [comment]
Merged. Thank you!

**Pull request merged**